identity/src/main/java/com/stripe/android/identity/ui/ConfirmationScreen.kt

```diff
@@ -33,86 +34,92 @@ import androidx.compose.ui.unit.sp
 import com.google.android.material.composethemeadapter.MdcTheme
 import com.stripe.android.identity.R
 import com.stripe.android.identity.networking.Resource
-import com.stripe.android.identity.networking.Status
 import com.stripe.android.identity.networking.models.VerificationPage
 import com.stripe.android.uicore.text.Html
 
 @Composable
 internal fun ConfirmationScreen(
-    verificationPageState: Resource<VerificationPage>?,
+    verificationPageState: Resource<VerificationPage>,
+    onError: (Throwable) -> Unit,
+    onComposeFinish: (VerificationPage) -> Unit,
     onConfirmed: () -> Unit
```

Comment:
Nit: Do we need the `onComposeFinish` parameter? Is there any `Modifier` that can be used instead?

Reply:
tracked in https://jira.corp.stripe.com/browse/IDPROD-5001